File: springboot-22/src/main/java/ro/jtonic/handson/cli/CliApp.java
package ro.jtonic.handson.cli;

import org.springframework.boot.Banner.Mode;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.util.StringUtils;

@SpringBootApplication
public class CliApp {

	public static void main(String[] args) {

	  String[] disabledCommands = {"--spring.shell.command.stacktrace.enabled=false"};
    String[] fullArgs = StringUtils.concatenateStringArrays(args, disabledCommands);

		new SpringApplicationBuilder(CliApp.class)
        .bannerMode(Mode.OFF)
        .run(fullArgs);
	}
}

File: handson-sb23/src/main/kotlin/ro/jtonic/handson/handsonsb23/CliApp.kt
package ro.jtonic.handson.handsonsb23

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.builder.SpringApplicationBuilder

@SpringBootApplication
class CliApp

fun main(args: Array<String>) {

  SpringApplicationBuilder(CliApp::class.java).run(*args)
}
